cmd/av/stack.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"emperror.dev/errors"
+	"github.com/spf13/cobra"
+	"strconv"
+)
+
+var stackCmd = &cobra.Command{
+	Use:   "stack",
+	Short: "managed stacked pull requests",
+}
+
+var stackBranchFlags struct {
+	// The parent branch to base the new branch off.
+	// By default, this is the current branch.
+	Parent string
+}
+var stackBranchCmd = &cobra.Command{
+	Use:   "branch <branch name>",
+	Short: "create a new stacked branch",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("branch name is required")
+		}
+		return errors.New("unimplemented")
+	},
+}
+
+var stackSyncFlags struct {
+	// Set the parent of the current branch to this branch.
+	// This effectively re-roots the stack on a new parent.
+	Parent string
+	// If set, incorporate changes from the trunk (repo base branch) into the stack.
+	// Only valid if synchronizing the root of a stack.
+	// This effectively re-roots the stack on the latest commit from the trunk.
+	Trunk bool
+	// If set, do not push to GitHub.
+	NoPush bool
+	// If set, we're continuing a previous sync.
+	// TODO:
+	// 	 we might not actually need this, we can probably detect that
+	//   a sync needs to be completed automagically and do the right thing.
+	Continue bool
+}
+var stackSyncCmd = &cobra.Command{
+	Use:   "sync",
+	Short: "synchronize all stacked branches after the current branch",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return errors.New("unimplemented")
+	},
+}
+
+var stackTreeFlags struct {
+	// Print the stack starting at this branch.
+	// If not set, we start at the base branch of the repository.
+	Root string
```

Comment:
seems unncessary. It's okay to define these things for future, but I really don't think we need to provide that many custom options in v1. We can just print the full stack.

Reply:
I'd prefer if we keep both these params out for now.